vulcan/lib/client/jsx/components/workflow/user_interactions/inputs/multiple_string.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {useState, useMemo, useEffect, useCallback, useRef} from 'react';
+import * as _ from 'lodash';
+
+import {InputBackendComponent} from './input_types';
+import TextInput from 'etna-js/components/inputs/text_input';
+
+function equalKeys(hash1: {}, hash2: {}): boolean {
```

Comment:
super nit: I think you can remove this function now, since it's cleaned up?

Also, if you want this to validate the annotation inputs as empty string is not allowed, you'll want to add that comparison to `selectors/workflow_selectors.ts/inputValueNonEmpty`, here: https://github.com/mountetna/monoetna/blob/master/vulcan/lib/client/jsx/selectors/workflow_selectors.ts#L112. Currently if you leave it as empty string, it would pass that along to the server -- not sure if you want that behavior or not?

Reply:
good nit!

For the validation, I'm not sure how widespread the `inputValueNonEmpty` use is, but I think we will want to allow selections of `''` in the future for other things.  Maybe we could pair briefly on the ideal way for me to set up only sometimes allowing `''`?